Cargo.toml

```diff
@@ -21,10 +21,10 @@ serde-big-array = "0.5.1"
 thiserror = "1.0.49"
 
 # plonky2-related dependencies
-plonky2 = "0.2.2"
+plonky2 = { git = "https://github.com/0xPolygonZero/plonky2.git", rev = "dc77c77f2b06500e16ad4d7f1c2b057903602eed" }
 plonky2_maybe_rayon = "0.2.0"
-plonky2_util = "0.2.0"
-starky = "0.4.0"
+plonky2_util = { git = "https://github.com/0xPolygonZero/plonky2.git", rev = "dc77c77f2b06500e16ad4d7f1c2b057903602eed" }
+starky = { git = "https://github.com/0xPolygonZero/plonky2.git", rev = "dc77c77f2b06500e16ad4d7f1c2b057903602eed" }
```

Comment:
We'll be doing a release this week normally, so can probably hold off until then and revert to using version tags

Reply:
In the end we didn't end up using the API changes on plonky2 right? With the new approach for dummy segment removal. Do we still need a version bump there? (It's not included in #276 as per discussion in the sync meeting).

Reply:
Sorry, I had missed this comment before: we still need methods such as conditional_assert_eq which are not on Plonky2 v.0.2.2.